test/pylib/cpp/boost/boost_facade.py

```diff
@@ -83,15 +91,19 @@ def list_tests(
                          case.endswith('*') and not case.strip().startswith('_')]
 
     def run_test(
-        self,
-        executable: Path,
-        original_name: str,
-        test_name: str,
-        mode: str,
-        file_name: Path,
-        test_args:Sequence[str] = (),
-        env: dict = None,
-    ) -> tuple[list[CppTestFailure], str] | tuple[None, str]:
+            self,
```

Comment:
Why changing the indentation of arguments? It doesn't seem that the run-test method really needs it here, does it?